src/web/backend/ldap/api.py

```diff
@@ -0,0 +1,78 @@
+from flask import Blueprint, jsonify, request, abort, current_app
+from flask.views import MethodView
+
+from edap import Edap, ObjectDoesNotExist
+
+from backend.utils import EncoderWithBytes
+from backend.settings import EDAP_USER, EDAP_DOMAIN, EDAP_HOSTNAME, EDAP_PASSWORD
+
+from .utils import get_config_divisions
+
+blueprint = Blueprint('divisions_api', __name__, url_prefix='/api/ldap/')
+blueprint.json_encoder = EncoderWithBytes
+
+
+class EdapMixin:
+
+    @property
+    def edap(self):
+        return Edap(EDAP_HOSTNAME, EDAP_USER, EDAP_PASSWORD, EDAP_DOMAIN)
+
+
+class DivisionsListViewSet(EdapMixin, MethodView):
+
+    def get(self):
+        """ Get divisions """
+        ldap_divisions = self.edap.get_divisions()
+        config_divisions = get_config_divisions()
+
+        divisions = {}
+
+        for machine_name, display_name in config_divisions.items():
```

Comment:
I just have a minor remark - please make a function/method `merge_divisions` that would perform the merge into the dictionary.
That way, the merge functionality can be unit-tested.

Reply:
You right, done